pallets/subtensor/src/lib.rs

```diff
@@ -1244,7 +1259,15 @@ pub mod pallet {
 			Self::do_sudo_set_serving_rate_limit( origin, serving_rate_limit )
 		}
 
+		// Sudo call for setting tx rate limit
 		#[pallet::weight((0, DispatchClass::Operational, Pays::No))]
+		pub fn sudo_set_tx_rate_limit( origin:OriginFor<T>, tx_rate_limit: u64 ) -> DispatchResult {  
+			Self::do_sudo_set_tx_rate_limit( origin, tx_rate_limit )
+		}
+
+		#[pallet::weight((Weight::from_ref_time(39_699_000 as u64)
```

Comment:
did you add weights for sudo_set_max_burn?

Reply:
let's remove the weights from this one, we don't need them for now.

Reply:
I don’t remember setting a weight here but it might’ve gotten added during a merge conflict resolution on accident.